metals/src/main/scala/scala/meta/internal/debug/DebugAdapter.scala

```diff
@@ -0,0 +1,6 @@
+package scala.meta.internal.debug
+import org.eclipse.lsp4j.debug.services.IDebugProtocolClient
+
+trait DebugAdapter {
```

Comment:
Is this trait necessary? It seems we replace it with `MetalsDebugAdapter` whereever `DebugAdapter` is used as a parameter type.

Reply:
If we want to maintain clear separation of concerns then yes, it is needed.
Otherwise `internal.eval` would have to be aware of the content of `nternal.metals`